api/src/main/java/com/toggl/api/network/PushServicesApi.kt

```diff
@@ -0,0 +1,14 @@
+package com.toggl.api.network
+
+import com.toggl.api.network.models.pushServices.FcmRegistrationToken
+import retrofit2.http.Body
+import retrofit2.http.DELETE
+import retrofit2.http.POST
+
+interface PushServicesApi {
+    @POST("me/push_services")
+    suspend fun subscribe(@Body totalsBody: FcmRegistrationToken)
+
+    @DELETE("me/push_services")
```

Comment:
```suggestion
    @HTTP(method = "DELETE", path = "me/push_services", hasBody = true)
```

Otherwise this throws `ava.lang.IllegalArgumentException: Non-body HTTP method cannot contain @Body.`